xmtp_mls/migrations/2024-11-05-191238_cache_wallet_addresses/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE wallet_addresses(
+    inbox_id TEXT NOT NULL,
+    wallet_address TEXT NOT NULL,
+    PRIMARY KEY (inbox_id, wallet_address)
```

Comment:
It'd probably be better to make `wallet_address` the primary key.

Reply:
And then add a unique index on `inbox_id`

Reply:
I'm not sure if we have a `wallet_address` connected to multiple `inbox_id`s, or it's always 1 `wallet_address` connected to and inbox?

Reply:
That's actually a great thing to bring up. Here's the XIP for that https://www.notion.so/ephemerahq/Multi-Wallet-Identity-XIP-ba30e6ddb30c45acb2dc4ff9eccae7ee

Ignore my unique index comment. I think the wallet PK would be good though.